hack/test-cmd.sh

```diff
@@ -147,6 +147,18 @@ openshift start \
   --etcd-dir="${ETCD_DATA_DIR}" \
   --images="${USE_IMAGES}"
 
+# validate config that was generated
+[ "$(oadm validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml 2>&1 | grep SUCCESS)" ]
```

Comment:
When is someone going to validate their config?  When they're working with config, or working with an unrelated action?  Having a root validate command in admin makes a lot of sense.  But if we're grouping a bunch of config commands together (and master-config would not be the noun, it would be server config) that lends itself in that direction.

I guess as an admin I would expect "oadm validate <thing>" where thing is probably master, node, config, network, storage, routing, registry, etc